internal/generator/ecc/template/point.go.tmpl

```diff
@@ -22,6 +22,9 @@ import (
 	{{else}}
 	"github.com/consensys/gnark-crypto/ecc/{{.Name}}/fp"
 	{{- end}}
+	{{- if eq .Name "bls12-381"}}
+	"github.com/consensys/gnark-crypto/ecc/{{.Name}}/hash_to_curve"
```

Comment:
Why only for bls12-381?

Reply:
We need hash to curve only for generating points not in the subgroup. It is implemented only for fuzzing purposes in geth. And as EVM supports only BN254 and BLS12-381, where BN254 is prime order, then it is implemented only for BLS12-381.